yaml/degoogle.yml

```diff
@@ -1676,13 +1676,13 @@ mobile applications:
       url: https://protonmail.com/support/knowledge-base/android/
       eyes: null
       text: >
-        Can be downloaded from the Play Store, but may work with one of the store replacements above (such as Yalp).
+        Can be downloaded from the Play Store, and work with other stores based on the google play store (such as Yalp, Aurora).
```

Comment:
Please change "and work with" to "and work**s** with", and capitalize "Google Play Store".